makerdao/governance-portal-v2
# Fix/poll order by enddate

### Link to Clubhouse story

https://app.clubhouse.io/dux-makerdao/story/169/order-polls-by-expiration-instead-of-creation-date

### What does this PR do?

Changes poll sorting to sort polls closest to expiring on the top

### Steps for testing:

Use this URL and view polls, closest to expiration should be at the top

https://governance-portal-v2-2759i1b6k-dux-core-unit.vercel.app/

<img width="797" alt="Screen Shot 2021-08-09 at 12 52 43 PM" src="https://user-images.githubusercontent.com/5225766/128696083-75ff2659-0d4a-4ac5-9ee0-02bd25e3cc21.png">
